app/javascript/components/batch_update/components/GeneInput.stories.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+
+import { storiesOf } from '@storybook/react'
+
+import GeneInput from './GeneInput'
+
+class GeneInputWrapper extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      value: this.props.initialValue,
+    }
+  }
+
+  render() {
+    let props = {
+      name: 'Fungible',
+      value: this.state.value,
+      onChangeValue: ({ name, value }) => {
+        console.log('yeah', name, value)
+        this.setState({ value })
+      },
+    }
+    return <GeneInput {...props} />
+  }
+}
+
+storiesOf('GeneInput', module)
```

Comment:
These weren't strictly necessary for this work, but I backfilled stories for a couple of components in order to work out the kinks / figure out the conventions.